Podverse/PVAuth.swift

```diff
@@ -170,26 +173,36 @@ class PVAuth: NSObject {
         }
     }
     
+    fileprivate func notifyLoggingIn() {
+        DispatchQueue.main.async {
+            NotificationCenter.default.post(name:NSNotification.Name(rawValue: kLoggingIn), object: self, userInfo: nil)
+        }
+    }
+    
+    fileprivate func notifyLoginFailed() {
+        DispatchQueue.main.async {
```

Comment:
same here. if the dispatch is on the main queue, the listeners shouldn't also need dispatching